Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -474,11 +493,22 @@ open class RouteOptions: DirectionsOptions {
             }
         }
         
+        if includesTollPrices {
+            params.append(URLQueryItem(name: CodingKeys.includesTollPrices.stringValue,
+                                       value: includesTollPrices.queryString))
+        }
+        
         return params + super.urlQueryItems
     }
     
 }
 
+extension Bool {
```

Comment:
Should it be marked as private? Or should it be moved to Extensions directory instead?

Reply:
This could be moved to a separate file for consistency with Array.swift and String.swift. `queryString` should be internal by default.

Reply:
Didn't want to draw too much attention to this small extension, just like `DateFormatter` or `LocationDistance` extensions we have nearby, since it is used just in this one file. Should've probably `fileprivate` it...